cfgov/unprocessed/css/enhancements/forms.less

```diff
@@ -341,15 +196,8 @@ select[multiple] {
     }
 }
 
-/* topdoc
-  name: Sets default styling for multi-selects
- */
 
-select[multiple] {
```

Comment:
Hmm I can't find this style in CF, are you sure it's not needed?

Reply:
https://github.com/cfpb/capital-framework/blob/canary/src/cf-forms/src/atoms/multiselect.less

Reply:
Ah. Okay if the markup is updated 👍

Reply:
Oh ok, I thought that happened during the v4 work. I'll search it out for this and the one below

Reply:
Hopefully is, but good time to check when the CSS is removed.

Reply:
We don't seem to be using multiselects outside of those that are controlled by the Multiselect scripting so I think this is safe. That said, I still discovered a bug [I've logged here](https://github.com/cfpb/cfgov-refresh/issues/3443)

Reply:
👍